euphoria-core/src/main/java/cz/seznam/euphoria/core/client/util/Pair.java

```diff
@@ -19,12 +19,12 @@
 import java.util.Objects;
 
 /**
- * Pair of any types. The pair has to fit in memory and is read-only.
+ * A pair, i.e. a tuple of two elements.
```

Comment:
ah, thanks for raising the topic about the pair's mutability! (the removal of this information was actually not intentional.) i fully agree, sooner or later we will have to think about the mutability to keep GC as low as possible. in general, it makes sense to have these types mutable; user provided types may be mutable as well and there's nothing special about the tuple classes - at least i don't see it.

however, to properly leverage the advantages of mutable tuples, we'll also have to think about object re-use in general within euphoria. this can be pretty far reaching. i'd say we tackle this topic as a separate issue.